src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,128 @@
+package duke;
+
+import duke.exception.InvalidCommandException;
+import duke.exception.InvalidDescriptionException;
+import duke.task.Task;
+
+import java.util.Scanner;
+
+public class Duke {
+    public static final String HALF_TAB = "  ";
+    public static final String BREAK = System.lineSeparator();
+    public static final String TAB = "    ";
+    public static final String BORDER = "    ____________________________________________________________";
+
+    private static TaskManager taskManager;
+
+    public static void main(String[] args) {
+        showLogo();
+        greet();
+        interact();
+        sayBye();
+    }
+
+    private static void interact() {
+        // create input scanner
+        Scanner in = new Scanner(System.in);
+
+        taskManager = new TaskManager();
+        while (in.hasNextLine()) {
+            String userInputLine = in.nextLine();
+
+            if (inputIsBye(userInputLine)) {
+                break;
+            } else if (inputIsList(userInputLine)) {
```

Comment:
for the methods which has only one line, it will be also okay to directly include it in the if statement, because I find it a little bit hard to read if we need to switch to different methods frequently.